packages/xml-views-validation/src/validators/attributes/none-unique-id.ts

```diff
@@ -0,0 +1,79 @@
+import { clone, flatMap, map, omitBy } from "lodash";
+import {
+  accept,
+  XMLAstVisitor,
+  XMLAttribute,
+  XMLDocument,
+  XMLToken,
+} from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { NoneUniqueIDIssue } from "../../../api";
+
+export function validateUniqueID(
+  xmlDoc: XMLDocument,
+  model: UI5SemanticModel
+): NoneUniqueIDIssue[] {
+  const idCollector = new IdsCollectorVisitor();
+  accept(xmlDoc, idCollector);
+  const idsToXMLElements = idCollector.idsToXMLElements;
+
+  const duplicatedIdsRecords = omitBy(idsToXMLElements, (_) => _.length > 1);
+  const issues: NoneUniqueIDIssue[] = flatMap(
+    duplicatedIdsRecords,
+    (currDupAttribsPerID, id) => {
+      return map(currDupAttribsPerID, (currDupAttrib, currIdx) => {
+        const currDuplicatedIdValToken = currDupAttrib.syntax.value;
+        const otherDuplicatedIdsAttribs = clone(currDupAttribsPerID).splice(
+          currIdx
+        );
+
+        return {
+          kind: "NoneUniqueIDIssue",
+          message: `None Unique ID value: "${id}" found.`,
```

Comment:
The term is "non-unique" or "nonunique", not "none-unique". Could you change at least the strings that use the term (the `kind` and `message`)?
https://en.wiktionary.org/wiki/non-unique#English

Reply:
done